src/Icon360.tsx

```diff
@@ -3,6 +3,7 @@ import { IconProps } from './types'
 
 const Icon360: React.FC<IconProps> = (props) => (
   <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" {...props}>
+    <title>{props.title === undefined ? 'Icon360' : props.title}</title>
```

Comment:
Instead of making the undefined check here, you can set a default value.

```
const Icon360: React.FC<IconProps> = ({title = 'Icon360', ...props}) => (
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" {...props}>	  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" {...props}>
    <title>{title}</title>
```

This will also make sure that you don't pass the `title` property to the `svg` tag, where it doesn't belong to.